fiwareskuld/expired_users.py

```diff
@@ -215,61 +261,3 @@ def get_trial_remaining_time(self, user):
         difference = date_object_new - date_object_old
 
         return trial_duration - difference.days
-
-    def __check_token(self):
-        """Check if the token is not blank"""
-        if self.token == "":
-            # We need to have a admin token in order to proceed.
-            raise ValueError("Error, you need to have an admin token. Execute the get_admin_token() method previously.")
-
-    def __check_credentials(self):
-        """Check if we have the credentials of the admin user"""
-        if self.__tenant is None or self.__username is None or self.__password is None:
-            # We need to have a admin token in order to proceed.
-            raise ValueError("Error, you need to define the credentials of the admin user. "
-                             "Please, execute the setCredentials() method previously.")
-
-    def getadmintoken(self):
-        """
-        Get the current admin token
-        :return: The Keystone admin token
-        """
-        return self.token
-
-    def gerlisttrialusers(self):
-        """
-        Get the list of trial users
-        :return: List of Trial users.
-        """
-        return self.listUsers
-
-    def getlistusers(self):
-        """
-        Global method that call the rest of internal one in order to recover the information of
-        the expired users.
-        :return: List of Expired Users id who have Trial role and expired, example:
-                    ['0f4de1ea94d342e696f3f61320c15253', '24396976a1b84eafa5347c3f9818a66a']
-        """
-        # Get the security token
-        self.get_admin_token()
-
-        # Get the list of Trial users
-        self.get_list_trial_users()
-
-        # Get the list of expired trial users
-        listusers = self.get_list_expired_users()
-
-        return listusers
-
-    def set_keystone_endpoint(self, serviceendpoint):
-        """ Set the service endpoint corresponding to the Keystone Service
-        :param serviceendpoint: The Keystone service endpoint
-        :return: None
-        """
-        self.KEYSTONE_ENDPOINT = serviceendpoint
-
-    def get_keystone_endpoint(self):
-        """ Get the Keystone service endpoint.
-        :return: The Keystone service endpoint
-        """
-        return self.KEYSTONE_ENDPOINT
```

Comment:
removing no needed tests. we don't need to manage tokes, osclient is taking care of it